index.html

```diff
@@ -46,9 +46,15 @@
             </div>
           </li>
           <li>
-            <div class="avatar small">
+            <div class=" profile avatar small">
```

Comment:
small ? text avatar image add a meaning full classname

Reply:
I want to keep additional (add-on) classes simple but I attached these classes to parent classes so these overlap with another.